v3.1/custom-rules/before-crs.dist/cleanlogs.conf

```diff
@@ -8,3 +8,5 @@
 # - allow request
 # - don't write an audit log for the request
 SecRule REMOTE_ADDR "^10\.1\.\d+\.1$" "phase:1,id:40000,nolog,allow,ctl:auditEngine=Off"
+SecRule REMOTE_ADDR "@ipMatch 127.0.0.1" "phase:1,id:40001,nolog,pass,t:none,ctl:ruleRemoveById=920300"
+SecRule REMOTE_ADDR "@ipMatch ::1" "phase:1,id:40002,nolog,pass,t:none,ctl:ruleRemoveById=920300"
```

Comment:
What's the difference between `ctl:auditEngine=Off` and `ctl:ruleRemoveById=920300` in this case? Shouldn't we do the former?

Reply:
The two new rules aren't really part of the health checks, right? We should probably separate them by a blank line and maybe even add an explaining comment. What do you think, @ccremer?

Reply:
I have no idea about modSecurity rules. I changed to `ctl:auditEngine=Off` for now.
Added an explanation comment.